### ISSUE TITLE
APO (only!) sourceId should not be expected to roundtrip back to fedora (roundtrip normalization)

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/dor-services-app#3337]
~~@andrewjbtw should APOs that have a sourceId~~
~~- retain it through roundtripping? OR~~
~~drop it (as we do now) and use normalization code to consider the original and roundripped XML equivalent?~~

We want normalization code for APOs only NOT to expect sourceIds for roundtripped Fedora XML.  In other words, for APOs only, normalize out sourceIds for `identityMetadata.xml`. Keep sourceId for items (and collections?)

examples:  bb466fn7951, bb540gx2176

```xml
Original XML for identityMetadata
<?xml version="1.0"?>
<identityMetadata>
  <sourceId source="Hydrus">adminPolicy-brg-2014-02-01T18:02:58.323Z</sourceId>
  <objectId>druid:bb466fn7951</objectId>
  <objectCreator>DOR</objectCreator>
  <objectLabel>APO for Visitations</objectLabel>
  <objectType>adminPolicy</objectType>
  <adminPolicy>druid:zw306xn5593</adminPolicy>
  <otherId name="uuid">159a768a-8b6b-11e3-98f3-0050569b3c6e</otherId>
  <tag>Project : Hydrus</tag>
  <tag>Remediated By : 4.6.6.2</tag>
</identityMetadata>

Normalized original XML for identityMetadata:
<?xml version="1.0"?>
<identityMetadata>
  <sourceId source="Hydrus">adminPolicy-brg-2014-02-01T18:02:58.323Z</sourceId>
  <objectId>druid:bb466fn7951</objectId>
  <objectCreator>DOR</objectCreator>
  <objectLabel>APO for Visitations</objectLabel>
  <objectType>adminPolicy</objectType>
</identityMetadata>

Roundtripped XML for identityMetadata:
<?xml version="1.0"?>
<identityMetadata>
  <objectId>druid:bb466fn7951</objectId>
  <objectCreator>DOR</objectCreator>
  <objectLabel>APO for Visitations</objectLabel>
  <objectType>adminPolicy</objectType>
</identityMetadata>
```

**Comment by andrewjbtw** [comment]
I'm not sure I understand the two options. We're eventually going to drop the sourceIds from the APOs through normalization, which I understand to be the second option. 

Does the first option mean dropping the sourceIds from the XML now? If so, I'm not ready to lose the sourceIds yet. I have one more task to do to record that data somewhere else before we remove it.

**Comment by ndushay** [comment]
@andrewjbtw this ticket will be to leave the sourceids in the XML, and to fix our roundtrip normalization code so it doesn't expect sourceIds in roundtripped XML for APOs (only).

**Comment by ndushay** [comment]
@andrewjbtw do collections have `sourceIds`?   If so, should they get the same treatment?  I assume we want to retain `sourceId` info for DROs.

**Comment by andrewjbtw** [comment]
I believe we are mapping sourceIds in collections because H2 is creating them, but I don't think they're being required. There was an H2 discussion about that but I wasn't closely involved.

We definitely want sourceIds in DROs (and agreements?), where they're required.

**Comment by justinlittman** [comment]
@ndushay Is this fixed by https://github.com/sul-dlss/dor-services-app/issues/3154?

**Comment by ndushay** [comment]
> @ndushay Is this fixed by #3154?

@justinlittman yes.  It appears that all sourceIds for APOs are source Hydrus.  Closing.
